scripts/download-ecosystem.mjs

```diff
@@ -0,0 +1,141 @@
+import axios from 'axios';
+import fs from 'node:fs/promises';
+import { scrubForFilename } from './helpers.mjs';
+
+const axiosHeaders = {
+  headers: {
+    'DOCK-API-TOKEN': `${process.env.DOCK_API_TOKEN}`,
+  },
+};
+
+const templatesToDownload = [{
+  templateId: process.env.NEXT_PUBLIC_BANK_IDENDITY_TEMPLATE_ID,
+  name: 'BANK_IDENDITY_TEMPLATE_ID'
+},
+  {
+    templateId: process.env.NEXT_PUBLIC_FORSUR_PROOF_TEMPLATE_ID,
+    name: 'FORSUR_PROOF_TEMPLATE_ID'
+  },
+{
+  templateId: process.env.NEXT_PUBLIC_FORSUR_VERIFICATION_PROOF_TEMPLATE_ID,
+  name: 'FORSUR_VERIFICATION_PROOF_TEMPLATE_ID'
+  },
+{
+  templateId: process.env.NEXT_PUBLIC_QUOTIENT_LOAN_PROOF_TEMPLATE_ID,
+  name: 'QUOTIENT_LOAN_PROOF_TEMPLATE_ID'
+  },
+{
+  templateId: process.env.NEXT_PUBLIC_URBANSCAPE_BANKBIO_TEMPLATE_ID,
+  name: 'URBANSCAPE_BANKBIO_TEMPLATE_ID'
+  },
+{
+  templateId: process.env.NEXT_PUBLIC_URBANSCAPE_CREDITSCORE_TEMPLATE_ID,
+  name: 'URBANSCAPE_CREDITSCORE_TEMPLATE_ID'
+  },
+];
+
+export async function downloadEcosystems() {
+  console.log('--- Downloading ecosystems from Certs ---');
+
+  const ecosystemsUrl = `${process.env.DOCK_API_URL}/trust-registries/`;
+      const ecosystemsResponse = await axios.get(ecosystemsUrl, axiosHeaders);
```

Comment:
Nitpick: extra tab here.

Reply:
fixed that one and the weird tabbing in the JSON above